src/ImageSharp/Formats/Webp/Lossy/Vp8Encoder.cs

```diff
@@ -71,10 +71,7 @@ internal class Vp8Encoder : IDisposable
         /// </summary>
         private int uvAlpha;
 
-        /// <summary>
-        /// Scratch buffer to reduce allocations.
-        /// </summary>
-        private readonly int[] scratch = new int[16];
+        private readonly bool alphaCompression;
 
         private readonly byte[] averageBytesPerMb = { 50, 24, 16, 9, 7, 5, 3, 2 };
```

Comment:
```suggestion
        // This uses C#'s optimization to refer to the static data segment of the assembly, no allocation occurs.
        private static ReadOnlySpan<byte> AverageBytesPerMb => new byte[] { 50, 24, 16, 9, 7, 5, 3, 2 };
```
While you're on it.

Need to update https://github.com/SixLabors/ImageSharp/blob/484bd77d8d2e0cca5bdbad6e528337fc21b4af94/src/ImageSharp/Formats/Webp/Lossy/Vp8Encoder.cs#L327 to new name too.